app.rb

```diff
@@ -0,0 +1,33 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/chitter.rb'
+
```

Comment:
Very nice, tidy and structured code. All is functioning well, tested via rspec and rackup.